src/templates/base/2019/base.html

```diff
@@ -184,79 +184,81 @@
 
   {% block footer %}
     <footer class="alt-bg">
-      <div class="home-logo">
-        <a class="navigation-logo" href="{{ url_for('home', year=year, lang=lang) }}">
-          {{ self.web_almanac_logo() }}
-        </a>
-      </div>
-      <hr>
-      <nav aria-label="{{ self.footer_title() }}" class="nav-items">
-        <ul>
-          {{ self.non_chapter_nav_links() }}
-          <li>
-            {{ year_switcher('footer') }}
-          </li>
-          <li>
-            {{ language_switcher('footer') }}
-          </li>
-        </ul>
-      </nav>
-      <div class="mobile-ha-social-media">
-        <a class="ha-logo" href="https://httparchive.org/" aria-labelledby="httparchive-logo-footer-mobile">
+      <div class="container">
```

Comment:
note, only change I made here is adding `<div class="container">` and wrap footer content in it.

Reply:
Thanks for the explanation - would really have confused without that!